plugins/identity/app/services/service_layer/identity_services/credential.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module ServiceLayer
+  module IdentityServices
+    # This module implements Openstack User API
+    module Credential
+
+      Creds = Struct.new(:access, :secret, :error)
+
+      def credential_map
+        @pcredential_map ||= class_map_proc(Identity::Credential)
+      end
+
+      def credentials(user_id, filter = {})
```

Comment:
This is just returning OS-EC2 credentials, why not calling the method `aws_credentials`? this would not lead to the confusion of which kind of credentials are being returned. Bdw. I couldn't find the method in the identity API documentation https://docs.openstack.org/api-ref/identity/v3/index.html#credentials, I would imagine that the API call your are using it is already removed from the latest version.

Reply:
changed the name to `AWSCredential` / `aws_credentials`